furusiya/util_functions.py

```diff
@@ -0,0 +1,46 @@
+import tdl
```

Comment:
I would rename this class. `util`/`helper` things quickly become dumping grounds for random things.

I would call it `targetting_functions`